packages/astro/src/core/build/index.ts

```diff
@@ -22,11 +22,6 @@ export interface BuildOptions {
 
 /** `astro build` */
 export default async function build(config: AstroConfig, options: BuildOptions = { logging: defaultLogOptions }): Promise<void> {
-	// polyfill WebAPIs to globalThis for Node v12, Node v14, and Node v16
-	polyfill(globalThis, {
-		exclude: 'window document',
-	});
-
```

Comment:
I'll take the other side: I'm not a huge fan of import side effects vs. a really explicit function call to apply a polyfill! I find it much easier to debug the later.

I don't feel too strongly, if I'm in the minority on this.

Reply:
That's also a fair take on it! I don't feel too strongly but you're right that a function call is more explicit.

Reply:
This was done initially because I was subclassing `Request`, so it needed to exist before I could do that. But I wound up refactoring when I realized I couldn't do that in a non-breaking way, so this code can probably be changed to how it was before. I'll see if I can do that.

Reply:
Ok, this is now explicitly done like before.